website/common/locales/en/contrib.json

```diff
@@ -31,8 +31,8 @@
   "modalContribAchievement": "Contributor Achievement!",
   "contribModal": "<%= name %>, you awesome person! You're now a tier <%= level %> contributor for helping Habitica. See",
   "contribLink": "what prizes you've earned for your contribution!",
-  "contribName": "Contributor",
-  "contribText": "Has contributed to Habitica (code, design, pixel art, legal advice, docs, etc). Want this badge? <a href='http://habitica.wikia.com/wiki/Contributing_to_Habitica' target='_blank'>Read more.</a>",
+  "contribName": "Contributor Badge",
```

Comment:
Can you remove the `Badge`? i don't think it should have been added :)

Reply:
No problem at all! I'll make that change today. :)

Reply:
Just wanted to let you know that I removed the `Badge` text! As a sidenote, I noticed that after fetching and rebasing from `develop` that the website no longer works for me when I visit localhost:3000. Instead, I get a JSON object with the message `"ENOENT: no such file or directory, stat '/habitica/dist-client/index.html'"`

I'm a little confused what happened there since I never deleted that file, but wasn't sure if this was maybe related to some of the other changes happening recently. Thanks!

Reply:
@cassidypignatello locally you should run two processes: `npm start` for the server on `localhost:3000` and `npm run client:dev` in another terminal for the client that you'll be able to access at `localhost:8080` and is the one you should use.

`/dist-client` is used to store files for the compiled version of Habitica but it's not regenerated automatically and should not be used when developing

Reply:
Ah, I completely forgot about running `npm run client:dev`, now everything's working as expected. Sorry about the confusion, still getting used to the dev workflow here. Thanks so much for clarifying!